src/components/review-card-ratings-only.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Grid, Card, Progress, Rating } from 'semantic-ui-react';
+
+const RatingsCard = (props) => {
+  const { reviewTitle, usefulProgress, workloadProgress, difficultyProgress,
+    enjoymentProgress, reviewDate, overallRating } = props;
+
+  return (
+
+    <div style={{ display: 'block', margin: '20px' }}>
+      <Card fluid raised>
+        <Card.Content>
+          {/* <div style={{ margin: '100' }}>
+            <Progress value={enjoymentProgress} total='5' progress='ratio' size='small' color='blue'/>
+          </div> */}
+
+          <Grid>
+            <Grid.Row className='review-row-ratings-only'>
+
+              <Grid.Column width={16}>
+
+                <div style={{ margin: '0px 260px 0px 0' }}></div>
+                <span style={{ float: 'right' }}>{reviewDate}</span>
+
+                <Card.Header className="header-test">
+                  <span>
+                    <h3 className='header-test' >{reviewTitle}</h3>
```

Comment:
A Card.Header should already be Header.. this is adding extra styling from the `h3` component and is unnecessary code. also you use `header-test` (a bad variable name) in both parent and child component (only use it once)